Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -91,6 +126,13 @@ class BRBViewController: UIViewController {
         
         state = newState
     }
+    
+    func showErrorAlert(error: String) {
+        let errorAlert = UIAlertController(title: error, message: "Unable to fetch new BRB account data. Please check your connection and try again.", preferredStyle: UIAlertControllerStyle.alert)
+        errorAlert.addAction(UIAlertAction(title: "Okay", style: UIAlertActionStyle.destructive, handler: nil))
```

Comment:
Why is this destructive? This is the (only) intended action. It should be `.default`.

Also, don't write out the whole class path if Swift can infer it. i.e.:

    errorAlert.addAction(UIAlertAction(title: "Okay", style: .default, handler: nil))